recurseFibonacci.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+//I cannot get this code to print the fibonacci sequence without having a zero case in the function and then it always prints zero as the first term
+int fibonacci(int N){
+	if (N==1)
+	{
+	return 1;
+	}
+	else
+	{
+	return fibonacci(N-1) + fibonacci(N-2);
```

Comment:
this doesn't really do much - where is the adding of the two previous elements being done?